rapidash.go

```diff
@@ -768,7 +770,7 @@ func (tx *Tx) RollbackDBOnlyUnlessCommitted() error {
 }
 
 func (tx *Tx) RollbackUnlessCommitted() error {
-	if !tx.isCommitted {
+	if !tx.IsCommitted() {
```

Comment:
If occurs error at committing to Cache, DB doesn't committed. 
In this case, I think that should try to rollback for DB .